requirements.txt

```
@@ -1,3 +1,3 @@
 pygame
-ppb-vector >= 1.0a1, < 2
+ppb-vector >= 1.0a2, < 2
```

This can be simplified by specifying a version [compatible with] 1.0a2:
```suggestion
ppb-vector ~= 1.0a2
```

[compatible with]: https://www.python.org/dev/peps/pep-0440/#compatible-release